drivers/usb/device/usb_dc_dw.c

```diff
@@ -540,11 +544,110 @@ static void usb_dw_handle_enum_done(void)
 }
 
 /* USB ISR handler */
-static void usb_dw_isr_handler(void)
+static inline void usb_dw_int_rx_flvl_handler(void)
```

Comment:
On which HW have you tested the changes?

Reply:
none, the only board supporting it is the intel_s1000_crb and I don't have one. Could you test yourself if you happen to own that board?

Reply:
It's OK for some code clean-up but I would not change the logic too much if we cannot test it.

Reply:
Logic itself is 100% same, it's only some re-factorization (moving some code into own function). I think that should be fine.